students/11th/JoeunLee/js/main.js

```diff
@@ -0,0 +1,32 @@
+const textarea = document.querySelector("textarea");
+textarea.addEventListener('input', changeColor);
+function changeColor() {
+    const replyUploadBtn = document.querySelector(".replyUploadBtn");
+    const textareaVal = textarea.value;
+    if(textareaVal) {
+        replyUploadBtn.removeAttribute("disabled");
+        replyUploadBtn.style.color="rgba(0,149,246,1)";
+    } else replyUploadBtn.style.color="rgba(0,149,246,.3)";
```

Comment:
```suggestion
    textareaVal && replyUploadBtn.removeAttribute("disabled");
    replyUploadBtn.style.color= textareaVal ? "rgba(0,149,246,1)" : "rgba(0,149,246,.3)";
```
1. 이렇게도 쓸 수 있습니다. 삼항연산자 참조해주세요
2. textareaVal보다 구체적인 이름으로 변수명 지정해주세요.